analysis/R/decode.R

```diff
@@ -326,15 +326,17 @@ Decode <- function(counts, map, params, alpha = 0.05,
 
   # Estimates from the model are per instance so must be multipled by h.
   # Standard errors are also adjusted.
-  fit$Total_Est <- floor(fit$Estimate)
-  fit$Total_SD <- floor(fit$SD)
-  fit$Prop <- fit$Total_Est / N
-  fit$LPB <- fit$Prop - 1.96 * fit$Total_SD / N
-  fit$UPB <- fit$Prop + 1.96 * fit$Total_SD / N
-
-  fit <- fit[, c("String", "Total_Est", "Total_SD", "Prop", "LPB", "UPB")]
-  colnames(fit) <- c("strings", "estimate", "std_dev", "proportion",
-                     "lower_bound", "upper_bound")
+  fit$estimate <- floor(fit$Estimate)
+  fit$proportion <- fit$estimate / N
+
+  fit$std_error <- floor(fit$SD)
+  fit$prop_std_error <- fit$std_error / N
+
+  fit$prop_low <- fit$proportion - fit$prop_std_error
```

Comment:
Thanks, I did this and merged to master.

Andy


On Tue, Jun 23, 2015 at 4:49 PM, Ilya Mironov <notifications@github.com>
wrote:

> In analysis/R/decode.R
> <https://github.com/google/rappor/pull/44#discussion_r33105579>:
>
> > -  fit$Total_Est <- floor(fit$Estimate)
> > -  fit$Total_SD <- floor(fit$SD)
> > -  fit$Prop <- fit$Total_Est / N
> > -  fit$LPB <- fit$Prop - 1.96 * fit$Total_SD / N
> > -  fit$UPB <- fit$Prop + 1.96 * fit$Total_SD / N
> > -
> > -  fit <- fit[, c("String", "Total_Est", "Total_SD", "Prop", "LPB", "UPB")]
> > -  colnames(fit) <- c("strings", "estimate", "std_dev", "proportion",
> > -                     "lower_bound", "upper_bound")
> > +  fit$estimate <- floor(fit$Estimate)
> > +  fit$proportion <- fit$estimate / N
> > +
> > +  fit$std_error <- floor(fit$SD)
> > +  fit$prop_std_error <- fit$std_error / N
> > +
> > +  fit$prop_low <- fit$proportion - fit$prop_std_error
>
> Multiply std_error by 1.96 to make the error bars be 95% CI.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/rappor/pull/44/files#r33105579>.
>